README.md

```diff
@@ -131,8 +131,11 @@ npm run dev
 
 <a name="migrate-pusher-to-ably"></a>
 ## Migrating from pusher/pusher-compatible broadcasters
-- The current Ably broadcaster is fully compatible with the [pusher](https://laravel.com/docs/9.x/broadcasting#pusher-channels), [old Ably Broadcaster](https://laravel.com/docs/9.x/broadcasting#ably) and [pusher compatible open source broadcasters](https://laravel.com/docs/9.x/broadcasting#open-source-alternatives).
-- The only difference is for **Leaving the channel** on client side, you should use [Ably Channel Namespaces](https://ably.com/docs/general/channel-rules-namespaces) conventions.
+The current Ably broadcaster is fully compatible with the [pusher](https://laravel.com/docs/9.x/broadcasting#pusher-channels), [old Ably Broadcaster](https://laravel.com/docs/9.x/broadcasting#ably) and [pusher compatible open source broadcasters](https://laravel.com/docs/9.x/broadcasting#open-source-alternatives). Please follow below steps to migrate properly from other broadcasters.
+
+**1. Leaving the channel**
```

Comment:
I think in terms of `echo` terminology, it's better to call it `leaving the presence channel`

Reply:
Hey, sorry this section is not about leaving channel presence, rather leaving the channel itself. In terms of ably terminology it's `unsubscribing to the channel`. 
So, it's possible with public, private or presence channels.

Leaving the channel (echo) = unsubscribe to channel (ably)

Reply:
Ah I see, thanks for explaining 👍

In that case we should change it to this so that it matches the [laravel documentation](https://laravel.com/docs/9.x/broadcasting#leaving-a-channel)
```suggestion
**1. Leaving a channel**
```